pennylane/math/__init__.py

```diff
@@ -62,8 +62,22 @@
 T = ar.numpy.transpose
 
 
+class NumpyMimic(ar.autoray.NumpyMimic):
+    """Subclass of the Autoray NumpyMimic class in order to support
+    the NumPy fft submodule"""
+
+    def __getattribute__(self, fn):
+        if fn == "fft":
+            return numpy_fft
+        return super().__getattribute__(fn)
+
+
+numpy_mimic = NumpyMimic()
+numpy_fft = ar.autoray.NumpyMimic("fft")
+
+
 def __getattr__(name):
-    return getattr(ar.numpy, name)
+    return getattr(numpy_mimic, name)
```

Comment:
The `NumpyMimic` class that is built into Autoray supports dispatching to the `linalg` and `random` submodules, but _not_ `fft`, so that has been added here.